### PR TITLE
LPS-132027

### PR DESCRIPTION
**Pull request by mlan0**
Related Issue: https://issues.liferay.com/browse/LPS-132027

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -410,7 +410,7 @@ the-setting-x-set-for-field-x-is-invalid=Setting ({0}) of field {1} is invalid.
 the-settings-x-set-for-field-x-are-invalid=Settings ({0}) of field {1} are invalid.
 the-small-image-file-could-not-be-saved=The small image file could not be saved.
 the-specified-redirect-url-is-not-allowed=The specified redirect URL is not allowed.
-the-specified-redirect-url-is-not-allowed-due-to-installation-settings.-add-x-into-the-property-x-to-fix-it=The specified redirect URL is not allowed due to installation settings. Add "{0}" into the property "{1}" to fix it.
+the-specified-redirect-url-is-not-allowed-due-to-installation-settings.-add-x-into-the-property-x-to-fix-it=The External Redirect URL {0} is not allowed. Set it in the portal property {1} to allow it.
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties:413]: @mlan0, can you please update the language key according to the new language value?

I mean rename to `the-external-redirect-url-x-is-not...`

**Pull request closed without merging**